packages/web-app-files/src/views/AppBanner.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="app-banner hide-desktop">
+    <a href="javascript:void(0);" class="app-banner-exit" aria-label="Close"></a>
+    <div
+      class="app-banner-icon"
+      style="
+        background-image: url('https://is1-ssl.mzstatic.com/image/thumb/Purple116/v4/87/4c/3c/874c3c90-d58d-837f-63fe-6a7dae6fd320/AppIcon-1x_U007emarketing-0-7-0-sRGB-85-220.png/460x0w.png');
+      "
+    ></div>
+    <div class="info-container">
+      <div>
+        <div class="app-title">ownCloud</div>
+        <div class="app-publisher">ownCloud GmbH</div>
+        <div class="app-additional-info">FREE - On the App Store</div>
+      </div>
```

Comment:
Needs to be read from theme/config

Reply:
sure... that's why i dont like draft requests...

Reply:
> sure... that's why i dont like draft requests...

What's the problem though? 😄  See it as a reminder that is nice to have, it's not meant as criticism.